.github/workflows/e2e_targeted.yaml

```
@@ -0,0 +1,78 @@
+name: e2e tests (targeted)
+
+on:
+  schedule:
+    - cron: '30 4 * * *'
+  workflow_dispatch:
+    inputs:
+      kubernetes-version:
+        description: 'Kubernetes version to test with'
+        required: true
+        default: 'v1.23.1'
+      istio-version:
+        description: 'Istio version to test with'
+        required: true
+        default: 'v1.12.2'
+      controller-image:
+        description: 'KIC Docker image to test with. The default "localhost:5000/kong/kubernetes-ingress-controller:ci" builds an image from the dispatch branch'
```

possibly slightly helpful rewording:

```suggestion
        description: 'KIC Docker image to test with. The default "localhost:5000/kong/kubernetes-ingress-controller:ci" builds an image from the branch which the workflow was dispatched for'
```